action/action-dsl/src/test/resources/actions2.groovy

```diff
@@ -40,3 +40,63 @@ action ('fixedTap') {
         phaseShifterFixedTap('NGEN_NHV1', 1)
     }
 }
+
+action ('deltaTap0') {
+    tasks {
+        phaseShifterDeltaTap('NGEN_NHV1', 0)
```

Comment:
Is `phaseShifterTap()` a better name? The "fixed" equivalent exist, so maybe we can avoid it (not sure of this proposition)
If you change this keyword, also change the corresponding tasks (java & groovy)